File: help/pt_pt/exercicios_da_ficha_3/Ficha03_001/Ficha03_001.js
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

//Main Function
function main() {
    document.write("Base \t\t: " );// print text
    base = parseFloat(prompt("Base \t\t: " ,"0.0") );
    
    document.write(base + "\n");// print input 
    document.write("Expoente \t: " );// print text
    expo = parseFloat(prompt("Expoente \t: " ,"0.0") );
    
    document.write(expo + "\n");// print input 
    document.write("\nPotência \t= " + Math.pow( base , expo ) );
    document.write("\nOperador \t= " + Math.pow( base , expo ) );
}

//JavaScript  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Mon, 2022 Sep 26 14:28

File: help/pt_pt/exercicios_da_ficha_3/Ficha03_001/Ficha03_001.java
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

import java.util.Scanner;

public class Ficha03_001 {

    static Scanner keyb = new Scanner(System.in);//keyboard

    //Main Function
    public static void main( String[] args) {
        System.out.print("Base \t\t: " );
        double base = Double.valueOf(keyb.nextLine());
        System.out.print("Expoente \t: " );
        double expo = Double.valueOf(keyb.nextLine());
        System.out.print("\nPotência \t= " + Math.pow( base , expo ) );
        System.out.print("\nOperador \t= " + Math.pow( base , expo ) );
    }

}
//Java  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Mon, 2022 Sep 26 14:28
